src/lib/converter/nodes/block.ts

```diff
@@ -114,7 +114,7 @@ export class BlockConverter extends ConverterNodeComponent<ts.SourceFile|ts.Bloc
             const resolved = context.resolveAliasedSymbol(symbol);
 
             // export declaration is always unique: no need of loop
-            const declaration = resolved?.declarations?.[0];
+            const declaration = resolved.declarations?.[0];
```

Comment:
This isn't correct. This is valid TS:

```ts
export interface A {
    a: number
}

export interface A {
    b: number
}
```

Reply:
You can do the same with namespaces

Reply:
I know we can declare 2 times the same interface, that result in a merges one, but can we really export 2 times over the same name something ?
In classic case (const) the second export erase the 1st one.

It means here we need to declare a unique merged interface ?

Reply:
Doh, that's what I get for looking at code with antihistamine in my system. You are correct.